docs/plugins/mapper-attachments.asciidoc

```diff
@@ -1,6 +1,9 @@
 [[mapper-attachments]]
 === Mapper Attachments Plugin
 
+deprecated[3.0.0, Mapper attachments has been deprecated by <<ingest-attachment>>]
+
```

Comment:
Actually, if we merge this just into master, then use this instead:

    deprecated[3.0.0,The `mapper-attachments` plugin has been replaced by the {plugins}/ingest-attachment.html[`ingest-attachment`] plugin]

If the `[ ]` link syntax inside the annotation doesn't work, then just drop the link